src/Native/Runtime/coreclr/gcinfodecoder.h

```diff
@@ -49,6 +49,9 @@ typedef ArrayDPTR(const uint8_t) PTR_CBYTE;
 #define SSIZE_T IntNative
 #define LPVOID void*
 
+typedef uint64_t DWORD64;
```

Comment:
It would be better to update gcinfodecoder to use the existing types:

`DWORD64` -> `SIZE_T`
`ULONGLONG` -> `TADDR`

Add casts if necessary - like it is done for other platforms.

Reply:
That is, indeed, a much better solution.